public/data/processes.json

```diff
@@ -7942,5 +7942,95 @@
     "elec_MJ": 1.1556e1,
     "waste": 8.7e-2,
     "alias": null
+  },
+  {
+    "name": "Impression pigmentaire, procédé représentatif, traitement moyen des eaux usées",
+    "info": "Textile > Ennoblissement > Impression",
+    "unit": "m2",
+    "source": "Base Impacts",
+    "uuid": "710987ca-f483-4f1c-9122-38b620f4062b",
+    "impacts": {
+      "acd": 2.11988e-4,
+      "cch": 6.20522e-2,
+      "etf": 0.0,
+      "fru": 1.35876,
+      "fwe": 6.81551e-5,
+      "htc": 0.0,
+      "htn": 0.0,
+      "ior": 9.38064e-4,
+      "ldu": 2.9734e-1,
+      "mru": 2.87832e-7,
+      "ozd": 7.62257e-9,
+      "pco": 2.53113e-4,
+      "pma": 4.09434e-9,
+      "swe": 1.38926e-3,
+      "tre": 2.57148e-4,
+      "wtu": 0.0
+    },
+    "heat_MJ": 7.25,
+    "elec_pppm": 0.0,
+    "elec_MJ": 4.56,
+    "waste": 0.0,
+    "alias": "printing-pigment"
+  },
+  {
+    "name": "Impression fixé-lavé, procédé représentatif, traitement moyen des eaux usées",
+    "info": "Textile > Ennoblissement > Impression",
+    "unit": "m2",
+    "source": "Base Impacts",
+    "uuid": "0810b8e5-a3d9-4607-bccc-83d112573260",
+    "impacts": {
+      "acd": 6.92673e-4,
+      "cch": 3.77105e-2,
+      "etf": 0.0,
+      "fru": 6.24761e-1,
+      "fwe": 9.07767e-5,
+      "ior": 1.43026e-3,
+      "htc": 0.0,
+      "htn": 0.0,
+      "ldu": 2.34857e-1,
+      "mru": 5.70619e-7,
+      "ozd": 3.79884e-9,
+      "pco": 1.85692e-4,
+      "pma": 5.63363e-9,
+      "swe": 1.42631e-3,
+      "tre": 1.33536e-4,
+      "wtu": 0.0
+    },
+    "heat_MJ": 8.72,
+    "elec_pppm": 0.0,
+    "elec_MJ": 5.22,
+    "waste": 0.0,
+    "alias": "printing-substantive"
+  },
+  {
+    "name": "Apprêt anti-tache, procédé représentatif, traitement moyen des eaux usées",
+    "info": "Textile > Ennoblissement > Appret chimique",
+    "unit": "kg",
+    "source": "Base Impacts",
+    "uuid": "63baddae-e05d-404b-a73f-371044a24fe9",
+    "impacts": {
+      "acd": 2.48038e-3,
+      "cch": 4.33475,
+      "etf": 0.0,
+      "fru": 6.79917,
+      "fwe": 3.4782e-5,
+      "ior": 9.57356e-3,
+      "htc": 0.0,
+      "htn": 0.0,
+      "ldu": 1.00838,
+      "mru": 9.57786e-5,
+      "ozd": 8.51755e-5,
+      "pco": 1.06633e-3,
+      "pma": 2.32444e-8,
+      "swe": 1.28288e-3,
+      "tre": 1.23719e-3,
+      "wtu": 0.0
+    },
+    "heat_MJ": 1.074e1,
+    "elec_pppm": 0.0,
+    "elec_MJ": 1.61,
+    "waste": 0.0,
+    "alias": null
```

Comment:
Is this last process used somewhere? Is it relevant to the "pigment" or "substantive" printing process?

Reply:
Nope, it's for the next card, do you want me to remove it?

Reply:
I'll keep it so it's already there.

Reply:
As long as it's here on purpose, and not a bad copy/paste, I don't care ;)